packages/flutter/test/cupertino/text_field_test.dart

```diff
@@ -3475,7 +3475,6 @@ void main() {
         from: tester.getTopRight(find.byType(CupertinoApp)),
         cause: SelectionChangedCause.tap,
       );
-      expect(state.showToolbar(), true);
```

Comment:
Prior to this pr the entire selection overlay will be rebuilt when the editableText rebuild with different selection.

I feel that is weird. if you have a selection with toolbar and handles on a textfield and you update the texteditingcontroller with different selection, you will see the selection gets moved to a new place with the handles but no toolbar.

After this PR, the toolbar will remain open when the selection moved to the new place.

That means we don't need to call the showtoolbar here in the test.